Source/Particles/PhysicalParticleContainer.H

```diff
@@ -363,6 +363,9 @@ protected:
     //radiation reaction
     bool do_classical_radiation_reaction = false;
 
+    // A flag to enable saving of the previous timestep positions
+    bool m_save_prev_pos = false;
```

Comment:
My own preference is to have names spelled out more, so for example `m_save_prev_position` or the long winded `m_save_previous_position` or `m_save_old_position`. I find code easier to read if I'm next having to constantly work out what the abbreviations stand for.

Reply:
No problem; I've changed the name to `m_save_old_position`.